docker/app/Dockerfile

```diff
@@ -47,37 +57,88 @@ ENV LANG=en_US.UTF-8
 ENV LANGUAGE=en_US.UTF-8
 # Tell Puppeteer to skip installing Chrome. We'll be using the installed package.
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
-ENV PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium-browser
+ENV PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium
 ENV GROVER_NO_SANDBOX=true
-ENV CHROMIUM_PATH=/usr/bin/chromium-browser
+ENV CHROMIUM_PATH=/usr/bin/chromium
 
 # Allow tls connection to database with verification
 COPY docker/app/us-east-1-bundle.pem  /etc/ssl/certs/us-east-1-bundle.pem
 # Deprecated and soon to expire
 COPY docker/app/rds-combined-ca-bundle.pem /etc/ssl/certs/rds-combined-ca-bundle.pem
 
-SHELL [ "/bin/ash", "-o", "pipefail", "-c" ]
-RUN addgroup -g ${GROUP_ID} app-user || echo ok \
-  && grep ":${GROUP_ID}:" /etc/group | cut -f1 -d: > /.group_name \
-  && adduser -u ${USER_ID} -G "$(cat /.group_name)" -h /home/app-user -D app-user \
+RUN groupadd -g ${GROUP_ID} app-user \
+  && useradd -l -u ${USER_ID} -g app-user -s /bin/bash -d /home/app-user -m app-user \
   && mkdir /app /bundle \
-  && chown app-user:${GROUP_ID} /bundle /app
-
-RUN apk update \
-  && apk upgrade --no-cache \
-  && apk add --no-cache \
-  nodejs yarn \
-  aws-cli \
-  tzdata \
-  git \
+  && chown app-user:app-user /bundle /app
+
+# Add PostgreSQL official repository, NodeJS and Yarn repositories
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends curl ca-certificates lsb-release gnupg \
+  && sh -c 'echo "deb https://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+  && curl -fsSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor -o /etc/apt/trusted.gpg.d/postgresql.gpg \
+  && curl -fsSL https://deb.nodesource.com/setup_18.x | bash -s -- -y \
+  && apt-get update \
+  && apt-get remove -y nodejs \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get upgrade -y \
+  && apt-get install -y --no-install-recommends \
+  awscli \
   bash \
-  linux-headers \
-  icu icu-dev \
-  curl libcurl curl-dev \
-  libmagic file-dev file libffi-dev yaml-dev \
-  build-base libxml2-dev libxslt-dev postgresql-dev tmux postgresql shared-mime-info \
-  tar xz \
-  && git config --global --add safe.directory /app
+  build-essential \
+  chromium \
+  chromium-common \
+  dbus \
+  dbus-user-session \
+  file \
+  fonts-dejavu \
+  fonts-droid-fallback \
+  fonts-freefont-ttf \
+  fonts-liberation \
+  freetds-dev \
+  git \
+  gnupg-agent \
+  gpg \
+  imagemagick \
+  libcurl4 \
+  libcurl4-openssl-dev \
+  libffi-dev \
+  libfreetype-dev \
```

Comment:
Our pocketbook, build times, and boot times will thank us if we can reduce this set of packages. Maybe Dave could take a pass?

Reply:
removed, pretty sure we don't need these, I'm not seeing any PDF generation in ere